README.md

```diff
@@ -58,8 +58,11 @@ to build.
 1. Install the prerequisites: `npm install`
 1. If you are using the `develop` branch of vector-web, you will probably need
    to rebuild one of the dependencies, due to
-   https://github.com/npm/npm/issues/3055: `(cd node_modules/matrix-react-sdk
-   && npm install)`
+   https://github.com/npm/npm/issues/3055:
+   ```
```

Comment:
the formatting is broken here: https://github.com/vector-im/riot-web/blob/5258bf9c8455e97ce59e82150c51e31c0c681a44/README.md#building-from-source